cve_bin_tool/cli.py

```diff
@@ -208,8 +212,27 @@ def scan_file(self, filename):
                         % (filename, result["is_or_contains"], modulename, version)
                     )
                     if found_cves.keys():
-                        self.logger.info("Known CVEs in version " + str(version))
-                        self.logger.info(", ".join(found_cves.keys()))
+                        if output_csv_file:
```

Comment:
This might be a good time to refactor the output code (that's why this was listed as part of a gsoc project and not something we'd planned to just do a quick fix on) so that you can have a function that goes something like `cve_output("csv", cves)`  or `cve_output("console", cves)` and not have the mess of output that we have now.